hknweb/exams/templates/exams/index.html

```diff
@@ -21,12 +34,21 @@
 
 {% block content %}
 <div id="actual-contents">
+    <center>
+        <select id="selectSemestercourses" style="width: 300px">
+            <option selected disabled="true">--- Search for Course ---</option>
```

Comment:
This appears twice; let's have it only appear on the search button and not as a disabled option?
![image](https://user-images.githubusercontent.com/46059916/91670449-75e20580-ead2-11ea-8f17-94e6615d6d8a.png)

Reply:
Done